helm/hpcc/templates/dali.yaml

```diff
@@ -56,16 +56,17 @@ spec:
     spec:
       {{- include "hpcc.placementsByPodTargetType" (dict "root" $ "pod" $dali.name "type" "dali") | indent 6 }}
       serviceAccountName: "hpcc-default"
+      {{- $sashaPlanes := list }}
+      {{- if not ( $sashaServices.disabled ) -}}
+      {{- range $sashaName, $_sasha := $dali.services -}}
+      {{- $sasha := ($_sasha | default dict) -}}
+      {{- if and (not $sasha.disabled ) ( $sasha.storage) }}
```

Comment:
extra spaces before ) and after ( on one side

Reply:
not new, but will fix.